diff --git a/Cargo.toml b/Cargo.toml
index 34c9f20..41d0b45 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -119,7 +119,8 @@ exclude = [
 
 [features]
 ## Default features of crosvm. This selection is somewhat arbitrary for historical reasons.
-default = ["audio", "balloon", "config-file", "document-features", "gpu", "qcow", "usb", "libvda-stub", "slirp"]
+#default = ["audio", "balloon", "config-file", "document-features", "gpu", "qcow", "usb", "libvda-stub", "slirp"]
+default = ["audio", "balloon", "config-file", "document-features", "qcow", "usb", "libvda-stub", "slirp"]
 
 ## Enables support for the Android [sparse image format](https://android.googlesource.com/platform/system/core/+/HEAD/libsparse/sparse_format.h)
 ## in the block device.
diff --git a/gpu_display/build.rs b/gpu_display/build.rs
index 86fc8b9..7a0c6bc 100644
--- a/gpu_display/build.rs
+++ b/gpu_display/build.rs
@@ -110,5 +110,6 @@ fn main() {
     }
     build.compile("display_wl");
 
-    println!("cargo:rustc-link-lib=dylib=wayland-client");
+//    println!("cargo:rustc-link-lib=dylib=wayland-client");
+    println!("cargo:rustc-link-lib=wayland-client");
 }
diff --git a/third_party/minijail b/third_party/minijail
--- a/third_party/minijail
+++ b/third_party/minijail
@@ -1 +1 @@
-Subproject commit 4b0a832be96194c23a9c3dc76da10e344a6c69c5
+Subproject commit 4b0a832be96194c23a9c3dc76da10e344a6c69c5-dirty
diff --git a/tools/install-deps b/tools/install-deps
index 43bdc93..f378d9a 100755
--- a/tools/install-deps
+++ b/tools/install-deps
@@ -44,7 +44,6 @@ sudo apt-get install --yes --no-install-recommends \
     python3 \
     python3-argh \
     python3-pip \
-    python3-rich \
     qemu-system-x86 \
     rsync \
     screen \
@@ -53,7 +52,7 @@ sudo apt-get install --yes --no-install-recommends \
     wget
 
 # mdformat is not available as a debian package. Install via pipx instead.
-pipx install mdformat
+pipx install mdformat --force
 pipx inject mdformat mdformat-gfm mdformat-footnote
 pipx ensurepath
 
